content/en/docs/instrumentation/net/getting-started/_index.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Getting Started
+weight: 2
+---
+
+OpenTelemetry .NET is unique among OpenTelemetry implementations, as it is
+integrated with the .NET `System.Diagnostics` library. At a high level, you can
+think of OpenTelemetry for .NET as a bridge between the telemetry available
+through `System.Diagnostics` and the greater OpenTelemetry ecosystem, such as
+OpenTelemetry Protocol (OTLP) and the OpenTelemetry Collector.
+
+## Installation
+
+OpenTelemetry is available as a [NuGet package][1]. Install it with your
+preferred package manager client.
+
+## .NET Core
+
+If using .NET Core, you can follow the pages in this section to manually
+instrument Logs, Traces, and Metrics.
+
```

Comment:
not sure the best way to do this, should the contents of `traces.md` just be placed here or is keeping them separate and informing about the 3 pages here is better